e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -180,6 +180,40 @@ test.describe('Example Imagery', () => {
         expect(resetBoundingBox.width).toEqual(initialBoundingBox.width);
     });
 
+    test('Image Pan/Zoom alt text is displayed (Linux)', async ({ page }) => {
```

Comment:
test.skip(process.platform != 'darwin', 'Image Plan / Zoom alt text is displayed (Linux)', async ({ page, browserName })